plugin.js

```diff
@@ -281,24 +286,64 @@ function processWorklets(t, path, processor) {
   }
 }
 
+const PLUGIN_BLACKLIST_NAMES = [
+  // '@babel/plugin-transform-destructuring', do not blacklist
```

Comment:
why is this here. if its not on a list, there is no point of adding it in a comment as if it "almost" was on a list